Analysis/Tasks/PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -149,8 +202,57 @@ struct femtoDreamProducerTask {
                             track.tofNSigmaEl(), track.tofNSigmaPi(), track.tofNSigmaKa(), track.tofNSigmaPr(), track.tofNSigmaDe());
         }
       }
-    }
-  }
+    } //end tracks loop
+
+    for (auto& v0 : fullV0s) {
+      auto postrack = v0.posTrack_as<aod::FilteredFullTracks>();
+      auto negtrack = v0.negTrack_as<aod::FilteredFullTracks>(); ///\tocheck funnily enough if we apply the filter the sign of Pos and Neg track is always negative
+      if (!v0Cuts.isSelectedMinimal(col, v0, postrack, negtrack)) {
+        continue;
+      }
+      v0Cuts.fillQA(col, v0); ///\todo fill QA also for daughters
+      auto cutContainerV0 = v0Cuts.getCutContainer(col, v0, postrack, negtrack);
+      if ((cutContainerV0.at(0) > 0) && (cutContainerV0.at(1) > 0) && (cutContainerV0.at(2) > 0)) {
+        int postrackID = v0.posTrackId();
+        int rowInPrimaryTrackTablePos = -1;
```

Comment:
I think it would be nice to make this a tiny function in the task since it is also used for the neg. child

Reply:
The tiny function is referred specifically to which part?
The definition of
int postrackID = v0.posTrackId();
int rowInPrimaryTrackTablePos = -1;
and then the loop over the tracks ID?

Reply:
exactly, something which takes all the necessary ingredients and return the proper ID